src/main/java/frc/robot/commands/ClimberGoTo.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.Constants;
+
+public class ClimberGoTo extends CommandBase {
+  /** Creates a new ClimberFullRetract. */
+  public double endpoint;
+  public double startpoint;
+  public ClimberGoTo(double Endpoint) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    endpoint = Endpoint;
+  }
+  
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    startpoint = RobotContainer.m_climberSubsystem.getCurrentPosition();
```

Comment:
You should NOT be trying to control a subsystem by calling back thru RobotContainer!  

The subsystems you need should be passed into the Command as inputs, saved in private members and then referenced via private members of the Command.

This applies to all the methods in this file.